labs/orion.md

```diff
@@ -0,0 +1,68 @@
+
+Orion **DOES NOT** have the following two blockchain properties:
+
+  - **Smart-Contracts**: A set of functions that manage data on the blockchain ledger. Transactions are invocations of one or more smart contract's functions.
+  - **Decentralization of Trust**: A permissioned setup of known but untrusted organizations each operating their own independent database nodes but connected together to form a blockchain network. As one node cannot trust the execution results of another node, ordering transactions must be done with a BFT protocol
+  and all transactions need to be independently executed on all nodes.
+
+Figure 1 presents the high level architecture of Orion.
+<img src="./images/orion-architecture.png" alt="Orion Architecture" width="800"/>
+
+Orion stores and manages the following five data elements:
+
+  1. **Users**: Storage of users' credentials such as digital certificate and their privileges. Only these users can access the Orion node.
+  2. **Key-Value Pairs**: Storage of all current/active key-value pairs committed by users of Orion node.
+  3. **Historical Key-Value Pairs**: Storage of all past/inactive key-value pairs using a graph data structure with additional metadata
+  such as the user who modified the key-value pair, all previous and next values of the key, transactions which have read or written to
+  the key-value pair, etc... It helps to provide a complete data lineage.
+  4. **Authenticated Data Structure**: Storage of Merkle Patricia Tree where leaf node is nothing but a key-value pair. It helps in
+  creating proofs for the existence of a key-value pair.
+  5. **Hash chain of blocks**: Storage of cryptographically linked blocks, where each block holds a set of transactions submitted
+  by the user along with its commit status, a summary of state changes in the form of Merkle Patricia's Root hash, etc... It helps in
+  creating a proof for the existence of a block or a transaction.
+
+The users of the Orion node can query these five data elements provided that they have the required privileges and
+also can perform transactions to modify active key-value pairs. When a user submits a transaction, the user receives a transaction receipt
+from the Orion node after the commit of a block that includes the transaction. The user can then store the receipt locally for performing
+client-side verification of proof of the existence of a key-value pair or a transaction or a block.
+
+# Initial Committers
+
+- https://github.com/cendhu
+- https://github.com/tock-ibm
+- https://github.com/gennadylaventman
+- https://github.com/C0rWin
+- https://github.com/dany-moshkovich
+
+# Sponsor
+
+- https://github.com/cendhu - Hyperledger Fabric Maintainer
+- https://github.com/C0rWin - Hyperledger Fabric Maintainer
+
+# Pre-existing repository
```

Comment:
Will these existing repositories be merged into a single orion repository? Or will there be an orion-server and an orion-sdk repo? Secondly, I do not see that there is a license associated with these repositories? Will this be re-licensed as Apache 2?

Reply:
Thanks, @tkuhrt, for the quick review.

1. We will move the existing repos to `orion-server` and `orion-go-sdk`. 
2. Most of the existing source files already have the following lines
```
// Copyright IBM Corp. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0
```
We will shortly add the Apache-2.0 License file to both repos.

Reply:
Sounds good.